README.md

```diff
@@ -1,71 +1,19 @@
-* just a small maintenance release for Hedy-1.0.0, which brings the missing upgrade possibility from previous releases
-* images for 
-  * Ubiquiti ERX SFP, TP-Link WR1043ND-v4
-  * support for RaspberryPi and RaspberryPi3 (compile yourself)
-* introduces interface "ffuplink" for a flexible configuration of the wired uplink
-* improved support of 802.11s mesh in LuCI
-* disabling of 802.11b wifi-rates by default
-* in preinit and failsafe it IP-address is 192.168.42.1/24 (like an unconfigured node)
-* the LAN-interface (br-lan) is not providing a default route on a unconfigured node, so manual reconfiguration is required when not using the assistent
-* the assistent will not offer the setup of a private AP anymore
-* when changing or initially confiuring VLANs (via LuCI) the interfacenames will also be changed (should fix #388)
-* a lot of security-fixes for linux and many packages (since Kathleen 0.3.0)
-
-## Features
-* based on [OpenWrt](https://openwrt.org/start) v17.01.4+ (lede-17.01 branch)
-  * Linux 4.4.116
-  * OLSR 0.9.0.3 (downgraded for BBB-VPN compatibility)
-  * B.A.T.M.A.N. 2016.5
-* custom package sets for different use cases (see the [Wiki-page](https://wiki.freifunk.net/Berlin:Firmware#Image-Typen))
-* new OLSR setup and configuration:
-  * SmartGateway for gateway selection (e.g. honors uplink speed)
-  * dynamic gateway plugin for uplink connectivity tests (gwcheck script removed)
-    on hosts: 85.214.20.141, 80.67.169.40, 194.150.168.168
-* a configuration [wizard](https://github.com/freifunk-berlin/packages-berlin/tree/master/utils/luci-app-ffwizard-berlin)
-  * starts after first boot and guides new users through the configuration of the router
-* monitoring of nodes through collectd
-* some well known network-interfaces
-  * "br-dhcp" for all client-traffic
-  * "wlan0-adhoc-2" and "wlan1-adhoc-5" for AdHoc-meshing between nodes in 2.4GHz and 5GHz
-  * "ffuplink" for all traffic sent via the uplink of the node-operator, in case "share traffic" is set up
-    * freifunk-policyrouting fixed/patched for ffuplink-interface
-* frei.funk as local DNS entry for your router
-  * you do not have to remember your IP to get access
-* change default lan ip address to 192.168.42.1/24
-  * avoids network collisions
-  * this is also used in OpenWrt failsafe
-* one dhcp network for APs and lan (bridged)
-* remove of autoipv6 and use of ULA ipv6 prefixes
-* default dns servers:
-  * 85.214.20.141 (FoeBud / Digital Courage)
-  * 80.67.169.40 (www.fdn.fr/actions/dns)
-  * 194.150.168.168 (dns.as250.net)
-  * 2001:4ce8::53 (as250)
-  * 2001:910:800::12 (french data network - http://www.fdn.fr/)
 
 ## Contact / More information
 
-The firmware wiki page is at: https://wiki.freifunk.net/Berlin:Firmware
+More user relevant information about the firmware are on the wiki page at: https://wiki.freifunk.net/Berlin:Firmware. There you can also find the 
+* [ReleaseNotes](https://wiki.freifunk.net/Berlin:Firmware/v1.0.1)
```

Comment:
It would be good to have a releases wiki page which links to all the releases itself.  Then we won't have to update this link anymore.

Reply:
There is a Releases-chapter (https://wiki.freifunk.net/Berlin:Firmware#Releases) to which we can link. this section can in turn link to the individual releases, but I was not willing to edit the official firmware-page already.

But I feel you also like the idea, of splitting this of.

Reply:
An overall releasepage is a good idea.
But having to update some minor lines in the documentation seems good in preparation of a release, this might help to review the whole document from release to release.